drizzlepac/hlautils/testutils.py

```diff
@@ -77,6 +79,11 @@ def compare_wcs_alignment(dataset, force=False):
         log.info("Default WCSNAME: {}".format(default_wcsname))
         alignment = {default_wcsname:extract_results(results)}
 
+        # Download the calibration reference files to ensure availability
+        ref_files = ref_from_image(imglist[0], ['IDCTAB'])
```

Comment:
ACS and WFC3 observations require the use of the IDCTAB, DGEOFILE and NPOLFILE reference files.